!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blendFront: 0.1
  blendBack: 0.0
  blendImages: false
  correction1: 0.0
  correction2: 0.0
  correction3: 0.0
  correction4: 0.0
  croppedHeight: 626.0
  croppedWidth: 636.0
  fovIn: 195.0
  fovOut: 90.0
  gridLines: false
  inputProjection: 1
  linearBlend: true
  outputProjection: 0
  pitch: 0.0
  roll: -90.0
  x: 1.0
  xCenter: 318.0
  y: 1.0
  yCenter: 320.0
  yaw: 0.0
  z: 1.0
state: []
